### PR TITLE
CSVS-47 Meaningless pair was replaced to structure

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

File: src/main/scala/com/bwsw/cloudstack/vault/server/common/ProcessingEvent.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package com.bwsw.cloudstack.vault.server.common
+
+import scala.concurrent.Future
+
+case class ProcessingEvent[T](process: Future[Unit], event: T)
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/vault/server/common/ProcessingEvent.scala]: case class ProcessingEventResult[T](event: T, result: Future[Unit])

**Review by placccebo — DISMISSED** [review]

File: src/main/scala/com/bwsw/cloudstack/vault/server/common/ProcessingEvent.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/vault/server/common/ProcessingEvent.scala]: ok

File: src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/util/CloudStackEventHandlerTestSuite.scala [hunk 2]
```diff
@@ -138,7 +138,7 @@ class CloudStackEventHandlerTestSuite extends FlatSpec with TestData with BaseTe
     val cloudStackEventHandler = new CloudStackEventHandler(controller)
 
     val resultFuture = cloudStackEventHandler.restartEvent(event) match {
-      case ProcessingEvent(future, successHandleEvent) =>
+      case ProcessingEventResult(successHandleEvent, future) =>
```
Comment (placccebo) [inline src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/util/CloudStackEventHandlerTestSuite.scala]: successHandleEvent -> handledEvent

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/util/CloudStackEventHandlerTestSuite.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/util/CloudStackEventHandlerTestSuite.scala]: ok

**Review by placccebo — APPROVED** [review]

**Pull request merged**